cli.js

```diff
@@ -1230,7 +1230,7 @@ async function init(packageJson, queries, options) {
     if (!playlist) return Promise.reject();
     logger.log(`\u27a4 Playlist Name: ${playlist.name}`);
     logger.log(`\u27a4 By: ${playlist.owner_name}`);
-    if (playlist.description) logger.log(`\u27a4 Description: ${entityDecode(playlist.description)}`);
+    if (playlist.description) logger.log(`\u27a4 Description: ${entityDecode(playlist.description.replace(/(<([^>]+)>)/gi, ""))}`);
```

Comment:
apply eslint suggestion

```suggestion
    if (playlist.description)
      logger.log(`\u27a4 Description: ${entityDecode(playlist.description.replace(/(<([^>]+)>)/gi, ''))}`);
```